markdown/2017-10-23.md

```diff
@@ -0,0 +1,179 @@
+
+<center>
+<div style="position:relative;width:50%">
+    ![Want want biscuit](https://i.imgur.com/VTZIP2c.png "I didn't know this biscuit existed until today. Interesting combination of salty and sweet, with a rice-y aftertaste. 8/10 would eat again.")
+</div>
+</center>
+
+---
+
+Hi all,
+
+Welcome back to yet another busy week full of events and coursework!
+
+Consistent with previous weeks, this week we bring you a whole _two_ events.
+First a workshop run by Next Jump tomorrow (Tuesday), and a G-Research coding
+challenge on Thursday. Also look forward to our first academic reading group
+which will be held next week.
+
+Last week's Welcome Bar Night was well attended, with 320 people in total. We
+hope you enjoyed the night! The Netcraft event also saw an impressive turnout
+of roughly 160 people. 
+
+Now, without further ado&hellip;
+
+---
+
+Our upcoming events:
+
+# Next Jump: Becoming a 10X Engineer
+
+<center>
+    ![Next Jump Event Header](https://i.imgur.com/k3JSBnW.png)
+</center>
+
+- **When:** **<font color="red">TOMORROW</font>** 18.30–20.30, 24th October 2017
+- **Where:** Huxley 341/342
+- **Link:** [Facebook](https://www.facebook.com/events/127663097980360/)
+
+Why should learning stop when you leave university? How can you can develop
+as a coder who leads? At Next Jump we call this becoming a 10X Engineer,
+which means developing software engineers who can not only build amazing
+products used by millions of people, but can lead global product teams of
+diverse backgrounds and skills.
+
+Hear software engineer James Hughes' story and how he grew to become a
+product lead in the UK just 1 year after graduation, helped develop coding
+curriculums for school children and gave talks to industry leaders to improve
+their company cultures, including Apple and the Ministry of Defense. In his
+talk, he will draw upon lessons he has learnt and helping you all to become
+10X engineers.
+
+# G-Research Coding Challenge
+
+<center>
+    ![G-Research Coding Challenge Header](https://i.imgur.com/ZuhEdEF.png)
+</center>
+
+- **When:** 18.30-21.00, 26th October 2017
+- **Where:** Huxley 341/342
+- **Link:** [Facebook](https://www.facebook.com/events/2207747869451881/)
+
+G-Research is a leading quantitative research and technology company. We
+apply scientific techniques to find patterns in large, noisy and real-world
+data sets, using the latest statistical and "big data" analysis methodologies
+to predict global financial markets.
+ 
+Please join us for our annual coding challenge where you will compete in
+teams in a series of progressively more difficult sentiment analysis
+challenges.
+
+This is a great chance to test your problem solving and coding skills and a
+chance to win a &#163;50 Amazon vouchers per team member (plus prizes for
+runners up!).
+ 
+To take part all you need is at least one laptop per team and have a
+development environment set up for C#, Java or Python and you must be able to
+provide your own internet access.
```

Comment:
this might be slightly confusing. @mauriceyap @ThomasBower feels on changing the blurb so things make sense?

Reply:
this is taken from the blurb they sent us

Reply:
that said the internet access thing seems a bit stupid to mention